implementations/python/provekit-lift-py-tests/src/provekit_lift_py_tests/editor_lsp.py

```
@@ -0,0 +1,528 @@
+
+
+# ---------------------------------------------------------------------------
+# The server.
+# ---------------------------------------------------------------------------
+
+
+def _uri_to_path(uri: str) -> str:
+    if uri.startswith("file://"):
+        rest = uri[len("file://"):]
+        # file:///abs/path -> /abs/path
+        return rest if rest.startswith("/") else rest
```

Ruff (0.15.15)

[warning] Useless `if`-`else` condition

(RUF034)